target_postgres/postgres.py

```diff
@@ -736,7 +743,7 @@ def add_column(self, cur, table_schema, table_name, column_name, data_type, defa
     def merge_put_schemas(self, cur, table_schema, table_name, existing_schema, new_schema):
         new_properties = new_schema['properties']
         existing_properties = existing_schema['properties']
-        for prop, json_schema in new_properties.items():
+        for prop, _ in new_properties.items():
```

Comment:
We can actually change this to:
`for prop in new_properties:`
(I realize this was like this before, but thought I'd mention it plus I know you are diving deep into python)